Library/Formula/plplot.rb

```diff
@@ -12,11 +12,14 @@ class Plplot < Formula
 
   option 'with-java'
 
+  patch :DATA
```

Comment:
Yup! Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!